ppr-ui/src/composables/mhrRegistration/useHomeOwners.ts

```diff
@@ -152,7 +152,7 @@ export function useHomeOwners (isMhrTransfer: boolean = false) {
     }
   }
 
-  const getGroupForOwner = (ownerId: string, isTransfer = false): MhrRegistrationHomeOwnerGroupIF => {
```

Comment:
Argument was not used

Reply:
There might be few other places we could remove this arguments, since we set it up at composable level.

Reply:
You bet, will have another look. 
The refactor came in with the rebase!